frontend-cert/js-projects/higher-order-functions/script.js

```diff
@@ -0,0 +1,48 @@
+function sortByYear(book1, book2) {
+  if (book1.release_year < book2.release_year) {
+    return -1;
+  }
+
+  if (book1.release_year < book2.release_year) {
```

Comment:
To match the instruction:

```suggestion
  if (book1.release_year > book2.release_year) {
```